_includes/_pagination.html

```diff
@@ -13,8 +13,19 @@
   <div class="load-more">
     <button class="btn {% if include.btn-type == 'white-outline' %}btn-white btn-outline{% else %}btn-blue{% endif %}" data-role="load-more" data-paginate="{{ include.collection }}">Load more{{ include.label | prepend: ' ' }}</button>
   </div>
-  {% endif %}
 
+  {% elsif include.n_of_total_pagination == "true" %}
+  <div class="text-center input-pagination">
+    <button class="btn btn-blue" id="pagination-previous-button"><<</button>
```

Comment:
- I converted the `<<` to `<`. I usually see `<` for go to the previous page and `<<` is used for page 1.
- I converted the button blue background to be just blue text. This will go with the submit button changes.
- I converted `<` to `&lt;` to separate it from the HTML tags.

```suggestion
    <button class="btn btn-link text-blue" id="pagination-previous-button">&lt;</button>
```